### ISSUE TITLE
Feature Request: PRESERVE type specialties

### ISSUE THREAD

**Issue by Emilinrose** [kwsch/pk3DS#490]
so this comes from a lovely little gen 3 randomizer called the "reasonable randomizer" that i loved.

The idea is that even though the pokemon are randomized, trainers with type specialties(swimmers, gym leaders, etc) will KEEP those specialties and just have random pokemon of that type.

So for example, Roxanne in gen 1 could have, say, an Onix and a Regirock, but she'd always have rock types.

another smaller idea is to expand the possible starter options to have ANY basic pokemon instead of just ones with 2 evolutions. i've had a lot of fun with stantler as a starter, for example.

**Comment by FerdinandoPH** [comment]
[
![Captura de pantalla 2021-08-02 110608](https://user-images.githubusercontent.com/29411787/127835259-954df30c-1f8b-4b8d-a6df-f004e142c233.png)
](url)
Isn't that an option already?

**Comment by Emilinrose** [comment]
No, those options give trainers, and gym trainers _**A**_ single type. It will pick one at random for them though. So for example, Roxanne gets dragon types for her gym, Brawly gets fire, etc.
